test/e2e/data/shared/v1alpha4/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+kind: Metadata
+releaseSeries:
+  - major: 0
+    minor: 4
+    contract: v1alpha4
```

Comment:
I thought there was discussion in CAPI office hours to keep clusterctl types at v1alpha3 for now